indico/modules/users/models/users.py

```diff
@@ -396,3 +443,45 @@ def unlink_to(self, obj, role):
         :param role: the role to use in the link
         """
         return UserLink.remove_link(self, obj, role)
+
+    def synchronize_data(self, refresh=False):
+        """
```

Comment:
yeah, and that line could be more concise.. "Synchronize user data" - just like in a git commit, details could go into the following lines.